AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_materialproviders/smithygenerated/aws_cryptography_keystore/client.py

```diff
@@ -0,0 +1,462 @@
+
+        :param input: Inputs for versioning a Branch Key.
+        """
+        return asyncio.run(
+            self._execute_operation(
+                input=input,
+                plugins=[],
+                serialize=_serialize_version_key,
+                deserialize=_deserialize_version_key,
+                config=self._config,
+                operation_name="VersionKey",
+            )
+        )
+
+    def get_active_branch_key(
+        self, input: GetActiveBranchKeyInput
+    ) -> GetActiveBranchKeyOutput:
+        """Get the ACTIVE version for a particular Branch Key from the Key Store.
+
+        :param input: Inputs for getting a Branch Key's ACTIVE version.
+        """
+        return asyncio.run(
+            self._execute_operation(
+                input=input,
+                plugins=[],
+                serialize=_serialize_get_active_branch_key,
+                deserialize=_deserialize_get_active_branch_key,
+                config=self._config,
+                operation_name="GetActiveBranchKey",
+            )
+        )
+
+    def get_branch_key_version(
+        self, input: GetBranchKeyVersionInput
+    ) -> GetBranchKeyVersionOutput:
+        """Get a particular version of a Branch Key from the Key Store.
+
+        :param input: Inputs for getting a version of a Branch Key.
+        """
+        return asyncio.run(
+            self._execute_operation(
+                input=input,
+                plugins=[],
+                serialize=_serialize_get_branch_key_version,
+                deserialize=_deserialize_get_branch_key_version,
+                config=self._config,
+                operation_name="GetBranchKeyVersion",
+            )
+        )
+
+    def get_beacon_key(self, input: GetBeaconKeyInput) -> GetBeaconKeyOutput:
+        """Get a Beacon Key from the Key Store.
+
+        :param input: Inputs for getting a Beacon Key
+        """
+        return asyncio.run(
+            self._execute_operation(
+                input=input,
+                plugins=[],
+                serialize=_serialize_get_beacon_key,
+                deserialize=_deserialize_get_beacon_key,
+                config=self._config,
+                operation_name="GetBeaconKey",
+            )
+        )
+
+    async def _execute_operation(
+        self,
+        input: Input,
+        plugins: list[Plugin],
+        serialize: Callable[[Input, Config], Awaitable[DafnyRequest]],
+        deserialize: Callable[[DafnyResponse, Config], Awaitable[Output]],
+        config: Config,
+        operation_name: str,
+    ) -> Output:
+        try:
+            return await self._handle_execution(
+                input, plugins, serialize, deserialize, config, operation_name
+            )
+        except Exception as e:
+            # Make sure every exception that we throw is an instance of ServiceError so
```

Comment:
Why wrap everything in `ServiceError`? Is it the language idioms?

Reply:
This is how Smithy-Python generates its clients